packages/ethtool/ethtool.spec

```diff
@@ -0,0 +1,30 @@
+Name:           %{_cross_os}ethtool
+Epoch:          2
+Version:        6.2
+Release:        1%{?dist}
+Summary:        Settings tool for Ethernet NICs
+License:        GPL-2.0-only AND GPL-2.0-or-later
+URL:            https://www.kernel.org/pub/software/network/ethtool/
+Source0:        https://www.kernel.org/pub/software/network/ethtool/ethtool-%{version}.tar.xz
+BuildRequires:  %{_cross_os}glibc-devel
+BuildRequires:  %{_cross_os}libmnl-devel
+
+%description
+%{summary}
+
+%prep
+%setup -n ethtool-%{version}
+
+%build
+%cross_configure
+%make_build
```

Comment:
This looks right, but for any new program we're cross-compiling, it's worth testing e.g. an arm64 AMI built on an x86_64 host to ensure that the binary's architecture is actually correct.

Reply:
I confirmed it works on x86_64 as well.